app/queries/servers/post.ts

```diff
@@ -190,6 +190,14 @@ export const queryPostsById = (database: Database, postIds: string[], sort?: Q.S
     return database.get<PostModel>(POST).query(...clauses);
 };
 
+export const queryPostsByType = (database: Database, type: string, sort?: Q.SortOrder) => {
+    const clauses: Q.Clause[] = [Q.where('type', type)];
+    if (sort) {
+        clauses.push(Q.sortBy('create_at', sort));
+    }
```

Comment:
We don't have any need for sorting at the moment, so let's remove it. If we need it in the future, we can add it then.